docs/QuickStart-Introduction.md

```diff
@@ -0,0 +1,32 @@
+---
+id: quick-start-introduction
+title: Introduction to React Native
+layout: docs
+category: Quick Start
+permalink: docs/quick-start-introduction.html
```

Comment:
What about for new pages? Should we have a more descriptive url? Or should something like this (if we kept it) be just `docs/introduction.html`?